src/pages/messages/components/NviCandidatePage.tsx

```diff
@@ -61,7 +87,36 @@ export const NviCandidatePage = () => {
       {registrationQuery.data && (
         <ErrorBoundary>
           <ErrorBoundary>
-            <PublicRegistrationContent registration={registrationQuery.data} />
+            <Box sx={{ position: 'relative' }}>
+              <PublicRegistrationContent registration={registrationQuery.data} />
+
+              {nextCandidateIdentifier && offsetNextCandidate && !isMobile && (
+                <IconButton
+                  component={Link}
+                  to={{
+                    pathname: getNviCandidatePath(nextCandidateIdentifier),
+                    state: offsetNextCandidateState,
+                  }}
+                  data-testid={dataTestId.tasksPage.nvi.nextCandidateButton}
+                  title={t('tasks.nvi.next_candidate')}
+                  size="small"
+                  sx={{
+                    bgcolor: 'white',
```

Comment:
Mulig å løse vis/skjul enda enklere uten en egen hook? (ikke sikkert svaret er ja, men syns det virker som en enklere løsning om det virker iaf
```suggestion
                    display: {xs: "none", sm: "block"},
                    bgcolor: 'white',
```

Reply:
Funker om jeg setter en `height`, men ikke med `aspectRatio` :thinking: